src/components/orders/crud/OrderDetail.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from 'react'
+import { Container, Row, Col } from 'react-bootstrap/'
+import DetailElement from '../../common/DetailElement'
+import OrdersApiService from '../../../utils/api/ordersApiService'
+import './OrderDetail.scss'
+
+export default function OrderDetail(props) {
+	const [order, setOrder] = useState(null)
+	const [customer, setCustomer] = useState(null)
+	const [productsList, setProductsList] = useState([])
+
+	useEffect(() => {
+		const fetchData = async () => {
+			const fetchedOrder = await OrdersApiService.getOrderById(props.match.params.id)
+
+			setOrder(fetchedOrder.data)
+			setCustomer(fetchedOrder.data.customer)
+			setProductsList(fetchedOrder.data.products)
+		}
+		fetchData()
+	}, [])
+
+	return (
+		<div className="order-detail">
+			{order && customer ? (
+				<Container>
+					<Row>
+						<Col>
+							<DetailElement header="Shipping Cost:" value={order.shippingCost} />
+						</Col>
+						<Col>
+							<DetailElement header="Shipping Company:" value={order.shippingCompany} />
+						</Col>
+						<Col>
+							<DetailElement header="Deadline:" value={order.deadline} />
+						</Col>
+					</Row>
+
+					<Row>
+						<Col>
+							<DetailElement header="Email:" value={order.email} />
+						</Col>
+						<Col>
+							<DetailElement header="Currency:" value={order.currency.toUpperCase()} />
+						</Col>
+						<Col>
+							<DetailElement header="Notes:" value={order.notes} />
+						</Col>
+					</Row>
+
+					<Row>
+						<Col>
+							<DetailElement header="Customer Name:" value={customer.name} />
+						</Col>
+						<Col>
+							<DetailElement header="Customer Email:" value={customer.email} />
+						</Col>
+						<Col>
+							<DetailElement header="VAT Identification Number:" value={customer.vatNumber} />
+						</Col>
+					</Row>
+
+					<Row>
+						<Col>
+							{order.confirmationSent ? (
+								<DetailElement header="Confirmation Sent?:" value="Sent!" />
+							) : (
+								<DetailElement header="Confirmation Sent?:" value="Not sent!" />
+							)}
+						</Col>
+						<Col>
+							{order.proformaSent ? (
+								<DetailElement header="Proforma Sent?:" value="Not sent!" />
+							) : (
+								<DetailElement header="Proforma Sent?:" value="Sent!" />
+							)}
+						</Col>
+						<Col>
+							{order.invoiceSent ? (
+								<DetailElement header="Invoice Sent?:" value="Not sent!" />
+							) : (
+								<DetailElement header="Invoice Sent?:" value="Sent!" />
+							)}
+						</Col>
+						<Col>
+							{order.settledPayment ? (
+								<DetailElement header="Settled Payment?:" value="Not sent!" />
+							) : (
+								<DetailElement header="Settled Payment?:" value="Sent!" />
+							)}
+						</Col>
+					</Row>
+
+					<Col>
```

Comment:
this is out of row, cols should always be within rows

Reply:
done